test/system/220-healthcheck.bats

```diff
@@ -268,4 +268,34 @@ Log[-1].Output   | \"Uh-oh on stdout!\\\nUh-oh on stderr!\\\n\"
     done
 }
 
+@test "podman healthcheck one run" {
+    local ctrname="c-h-$(safename)"
+    run_podman run -d --name $ctrname    \
+               --health-cmd "echo Hi"    \
+               $IMAGE /home/podman/pause >&3
+    cid="$output"
+
+    run_podman healthcheck run $ctrname
+    run_podman inspect $ctrname --format "{{.State.Health.Log}}" >&3
+    [ $(echo "$output" | grep -o "Hi" - | wc -l) -eq 1 ]
```

Comment:
Please do not use this form of testing in bats. I know it's what the bats people themselves use, but it makes for impossible-to-understand error messages.

Actually, gimme a few more minutes, I'm not sure I understand the purpose of this PR. I think it can be refactored further.

Reply:
I'm trying to figure out why the first time the `podma healthcheck run` command is run, two log entries are created. I am unable to reproduce this on the local machine.

Reply:
@Honny1 I haven't looked deeply what you are doing but when you run podman run it will create and start the systemd timer right away which fires of the first healtcheck run so if you then manually run podman healthcheck run again you get a second run, this of course depends on the systemd

e.g.
```
$ bin/podman run --name c1 --health-cmd "echo hi" -d quay.io/libpod/testimage:20240123 sleep 100 && bin/podman healthcheck run c1 && podman inspect c1 --format "{{json .State.Health}}"
11249e7dcbd93e62e2e3d6af75be1664cc7961e2c340e5df53ab932442ba03c4
{"Status":"healthy","FailingStreak":0,"Log":[{"Start":"2024-09-17T18:55:11.63495862+02:00","End":"2024-09-17T18:55:11.730240069+02:00","ExitCode":0,"Output":"hi\n"},{"Start":"2024-09-17T18:55:11.676075091+02:00","End":"2024-09-17T18:55:11.753759388+02:00","ExitCode":0,"Output":"hi\n"}]}
```